tokio/src/sync/mpsc/unbounded.rs

```diff
@@ -172,6 +172,56 @@ impl<T> UnboundedReceiver<T> {
         poll_fn(|cx| self.poll_recv(cx)).await
     }
 
+    /// Receives the next values for this receiver and extends `buffer`.
+    ///
+    /// This method returns the number of values populated in `buffer`.
```

Comment:
I think the word "populated" isn't a great fit.
```suggestion
    /// This method returns the number of values added to `buffer`.
```